fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -124,4 +124,21 @@
     <string name="translations_bottom_sheet_translate_to">Translate to</string>
     <string name="translations_bottom_sheet_negative_button">Not now</string>
     <string name="translations_bottom_sheet_positive_button">Translate</string>
+
+    <string name="translation_option_bottom_sheet_title">Translation Options</string>
+    <string name="translation_option_bottom_sheet_always_translate">Always offer to translate</string>
+    <string name="translation_option_bottom_sheet_always_translate_in_language">Always translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <string name="translation_option_bottom_sheet_translation_settings">Translation settings</string>
+    <string name="translation_option_bottom_sheet_about_translations">About translations in %1$s</string>
+
+    <string name="translation_settings_fragment_toolbar_title">Translations</string>
+    <string name="translation_settings_fragment_translation_preference">Translation preferences</string>
+    <string name="translation_settings_fragment_offer_to_translate">Offer to translate when possible</string>
+    <string name="translation_settings_fragment_always_download">Always download languages  in data saving mode</string>
```

Comment:
Could you check that there aren't two spaces here: ` languages  in`?

Reply:
Done